### PR TITLE
Fix subject import and relationship types

### PR DESCRIPTION
**Pull request by kehh**
* Remove extra subject types from the installation profile
* Make `Association` field wider and taller
* Remove `Type` from related subjects as all subjects have type of `Subject`
* Rename `Dates` on subject joins to `Relationship Dates`
* Make `subject` relationship type visible in related subject displays
* This relates to RWAHS-262 and RWAHS-147

### CODE REVIEW

File: profile/rwahs.xml [hunk 1]
```diff
@@ -304,55 +304,13 @@
         </label>
       </labels>
       <items>
-        <item idno="subject" enabled="1" default="0">
+        <item idno="Subject" enabled="1" default="0">
```
Comment (kehh) [inline profile/rwahs.xml]: I wasn't sure whether to capitalise this, but I did, which meant more changes elsewhere in the profile.

**Comment by leftclickben** [comment]
This looks good, but do you need to regenerate the displays now?  Otherwise the labels, etc are out of sync?  (And will potentially appear out of place in a future PR).

**Comment by kehh** [comment]
Pretty sure I did it in ef563d5, but will confirm a bit later.

**Comment by kehh** [comment]
(there are a couple of commits after that, but iirc I made the same changes in the derived displays or regenerated the code)

**Comment by kehh** [comment]
I've just checked this, and I had already done it.

**Comment by leftclickben** [comment]
Yeah, I see that now, sorry.  It was because of the way github shrinks the diff, it looked like all part of the UIs.

**Comment by kehh** [comment]
That could certainly get confusing!

**Pull request merged**